src/components/Search/FilterDropdowns/DropdownButton.tsx

```diff
@@ -51,12 +53,17 @@ function DropdownButton({label, value, viewportOffsetTop, PopoverComponent}: Dro
         vertical: 0,
     });
 
+    const [willAlertModalBecomeVisible] = useOnyx(ONYXKEYS.MODAL, {selector: (modal) => modal?.willAlertModalBecomeVisible, canBeMissing: true});
+
     /**
      * Toggle the overlay between open & closed, and re-calculate the
      * position of the trigger
      */
     const toggleOverlay = () => {
         setIsOverlayVisible((previousValue) => {
+            if (!previousValue && willAlertModalBecomeVisible) {
+                return previousValue;
```

Comment:
very minor but can we just return `false` here instead of `previousValue` (since `previousValue` will always be false, right?)? Would do the same thing but be a little easier to understand IMO.